aspnetcore/blazor/hybrid/static-files.md

```diff
@@ -176,9 +176,19 @@ In the following example component:
 
 ## Static assets limited to Razor components
 
-In scenarios where the app only uses static assets in Razor components, the static assets can be supplied from the app's web root (`wwwroot` folder).
+One or more `BlazorWebView` controls have a configured host file (:::no-loc text="HostPage":::), typically `wwwroot/index.html`. The :::no-loc text="HostPage"::: path is relative to the project. All static web assets (scripts, CSS files, images, and other files) that are referenced from a `BlazorWebView` are relative to its configured :::no-loc text="HostPage":::.
```

Comment:
I'm not sure I get why it's talking about "One or more BlazorWebView..." --> what is that in reference to?

Reply:
@guardrex In this case I think it's fine to just say "A `BlazorWebView` control has a configured host file...".